### PR TITLE
MapEditor: Double click to finish the creation of a complex map item

### PR DESCRIPTION
**Pull request by MaStr11**
The documentation is missing a description, of how to end a point sequence for a complex map item.

### CODE REVIEW

File: interface-elements-for-desktop/articles/map/map-editor.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Note that you can only edit the following map vector items:
 
 Create mode allows you to add new items to the map. Select one of the following symbols to create map items: ![Add Pushpin Button](../../images/map-editor-add-pushpin-button.png) ![Add Path Button](../../images/map-editor-add-path-button.png) ![Add Polyline Button](../../images/map-editor-add-polyline-button.png) ![Add Dot Button](../../images/map-editor-add-dot-button.png) ![Add Ellipse Button](../../images/map-editor-add-ellipse-button.png) ![Add Rectangle Button](../../images/map-editor-add-rectangle-button.png) ![Add Line Button](../../images/map-editor-add-line-button.png)
 
-You can add dots and pushpins by clicking on the required location. To create a complex map item, add points sequentially to form a map item. The following animation shows how to create a map path:
+You can add dots and pushpins by clicking on the required location. To create a complex map item, add points sequentially to form a map item. Use a mouse double click or press **Enter** to finish the sequence. The following animation shows how to create a map path:
```
Comment (MaStr11) [inline interface-elements-for-desktop/articles/map/map-editor.md:63]:
```suggestion
You can add dots and pushpins by clicking on the required location. To create a complex map item, add points sequentially to form a map item. Use a double-click or press **Enter** to finish the sequence. The following animation shows how to create a map path:
```

**Comment by MaStr11** [comment]
@serge-andreyev Can you assign a reviewer, to bring this to anyone's attention (there are two more PRs to review)? Thanks.

**Comment by KseniyaKuzmina** [comment]
Hi,
Thank you for your contribution to our documentation. We will update the topics to meet your suggestions.

**Comment by KseniyaKuzmina** [comment]
We updated the following help topics according to your suggestions:

* [Map Editor](https://devexpress.github.io/dotnet-eud/interface-elements-for-desktop/articles/map/map-editor.html)
* [Selection](https://devexpress.github.io/dotnet-eud/interface-elements-for-desktop/articles/map/selection.html)

**Comment by MaStr11** [comment]
Great!

**Pull request closed without merging**